### ISSUE TITLE
 Connect temperature sensor

### ISSUE THREAD

**Issue by Hcreak** [maximkulkin/esp32-homekit-camera#89]
I want to add DHT11 temperature and humidity sensor based on this. What do I need to do? Are there any good ideas? Thanks

**Comment by rottenhowler** [comment]
One idea is to code support for DHT11 into firmware. How about that?

**Comment by Hcreak** [comment]
> One idea is to code support for DHT11 into firmware. How about that?

That’s right. I mean the same, but I don’t know where to insert the code without conflict.
